src/status_im/contexts/onboarding/create_profile/view.cljs

```diff
@@ -10,10 +10,10 @@
     [react-native.platform :as platform]
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
+    [status-im.common.profile-picture-picker.view :as picture-picker]
```

Comment:
I noticed that in the rest of the files, the ns is aliased to `photo-picker`. I guess aliasing to `profile-picture-picker` should be fine.